.github/workflows/playground_examples_ci.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Playground Examples CI
+
+on:
+  push:
+    paths:
+     - .github/workflows/playground*
+     - playground/backend/**
+     - playground/infrastructure/**
+     - learning/katas/**
+     - examples/**
+     - sdks/**
```

Comment:
That seems reasonable. How long will this workflow run? If its not _too_ long (less than an hour), I'd probably just run it on all commits to master.

Changes to the SDK theoretically should never break TOB (we want to avoid breaking changes like that), so having a clear signal on which commit broke it would be very valuable.

Reply:
It takes around 30 minutes, 10 minutes for each SDK
AFAIU commits are usually pushed to non-master branch, and only when PR is merged there's push to master. In order for CI to confirm there's no breaking changes in PR, it needs to not be limited to master branch pushes.

Reply:
There's a follow-up PR for Example CD which is meant to be run on PR merge: https://github.com/akvelon/beam/pull/275
It ensures that examples can run on the current playground backend

So, running CI too would be pointless.
There's a possibility for an optimization:
playground examples are spread across `./sdks`, `./learning/katas`, and `./examples` subdirs. There are many other files which could trigger CI. We may check if a particular push amends any examples of a given SDK, and only then proceed to heavy steps.

We could then try to optimize it even further, if it turns out too heavy

Reply:
That seems reasonable to me. I do like the idea of adding this validation